Monika After Story/game/script-topics.rpy

```diff
@@ -15131,16 +15131,24 @@ init 5 python:
     )
 
 label monika_dying_same_day:
-    m 1euc "[player], have you ever heard of couples wishing to die on the same day?"
-    m 3rksdla "...Gosh, that sounds kinda dark, doesn't it?"
-    m 3rfd "Before I met you, I thought the idea was completely absurd!"
-    m 1etc "Why would someone give up on living a long and happy life just because their partner's life was dramatically cut short?"
-    m 3rfd "It would be irrational and impulsive to do so!"
-
-    if mas_isMoniEnamored(higher=True):
-        m 2ekc "...But with you by my side and all the time I've spent here with you...{w=0.3}I can kind of see it now."
+    if mas_event_mailbox.is_monika_initiated_mode():
+        m 1euc "[player], have you ever heard of couples wishing to die on the same day?"
+        m 3rksdla "...Gosh, that sounds kinda dark, doesn't it?"
+        m 3rfd "Before I met you, I thought the idea was completely absurd!"
+        m 1etc "Why would someone give up on living a long and happy life just because their partner's life was dramatically cut short?"
+        m 3rfd "It would be irrational and impulsive to do so!"
+        if mas_isMoniEnamored(higher=True):
+            m 2ekc "...But with you by my side and all the time I've spent here with you...{w=0.3}I can kind of see it now."
+        else:
+            m 2ekc "...But after meeting you, I think I can kind of see it now."
     else:
-        m 2ekc "...But after meeting you, I think I can kind of see it now."
+        m "You know, I think I could understand couples wishing to die together, on the same day."
+        m "I was really sceptical about this before, thinking that it's such an irrational idea to follow your loved one there."
```

Comment:
Pretty sure that if you don't set an expression here, these 2 lines will keep using either the `idle` exp, or the one that was set during the previous topic.

Reply:
How embarrassing.

Reply:
This still needs to be fixed.